cyences_app_for_splunk/bin/cyences_mvintersection.py

```diff
@@ -0,0 +1,44 @@
+import sys
```

Comment:
Just add **import cs_imports** for the splunklib import

Reply:
Done